### ISSUE TITLE
Validate Voltage Ratings

### ISSUE THREAD

**Issue by marcusmueller** [dkozel/noisesource-psu#10]
Love the output filter! I don't see any reasonable argument for it ever resonating, but, still, 28+1 V is ever so slightly too close to the 35 V of your 10 µF MLCCs,  C11 and C12, for my overly anxious mind.

**Comment by dkozel** [comment]
Agreed. I'm going to review the BOM for the next rev, looking to cost save and improve safety margins/ratings where possible.
